assignments/array-methods.js

```diff
@@ -55,29 +55,50 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+// 
+let fullName = [runners.first_name + runners.last_name];
+
+runners.forEach(runners => console.log(fullName));
+
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+let allCaps = runners.map(x => x.first_name.toUpperCase());
 console.log(allCaps); 
 
+
+
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+let largeShirts = shirt_size.filter(size => size = L);
```

Comment:
inside the filter, you may need use quotes around the "L" because it is string. Otherwise, the JS engine will t.hink you are trying to declare a variable. The assignment operator, "=",  should be === operator because we are comparing values and not assigning.